src/map/map.h

```diff
@@ -900,6 +900,8 @@ struct map_data {
 		unsigned pairship_startable : 1;
 		unsigned pairship_endable : 1;
 		uint32 noviewid; ///< noviewid (bitmask - @see enum equip_pos)
+		int32 nostorage;
+		int32 nogstorage;
```

Comment:
Better have it as
```c
unsigned nostorage : 2;
```
So that it would only allocate 2 bit (suitable for value till 3)

Reply:
yes here better use bit fields, and put new flags before noviewid for pack new fields into existing bitfields